src/game/Ship.java

```diff
@@ -57,6 +60,18 @@ void addCrewmember(Person p) {
 
 
     public int getCrewCount(){ return crew.size(); }
+    public List<Person> getCrew() {return crew; }
+    public boolean crewIsAlive() {
+        boolean crewAlive = false;
+        for(Person p : crew)
+        {
+            if(p.getHealthPoints() > 0) {
+                crewAlive = true;
+            }
+        }
```

Comment:
Taki zapis jest ok, ale zamiast fora można to zrobić streamem (jest to bardziej czytelne, poza tym dzięki lazy-evaluation można zaoszczędzić sporo na wydajności kiedy program ma wykonywać daną operację wiele razy i nie za każdym razem dochodzić do końca kolekcji w pętli).
Alternatywny zapis ze streamem:
```
public boolean crewIsAlive() {
        return crew.stream().anyMatch(person -> person.getHealthPoints() > 0);
    }
```